### DISCUSSION TITLE
if zed could debug code,i'll remove vscode now.

### DISCUSSION THREAD

**Discussion by lyonxh** [zed-industries/zed#10712] [Feature Requests]
and,i hope a windows version can be released

**Comment by skylovescoffee** [comment]
I second this, with complex projects I cannot do meaningful work without a debugger, it's the only thing stopping me from using zed full time

**Comment by SomeoneToIgnore** [comment]
It will, eventually, as it is planned and there are related issues for tracking things:
* https://github.com/zed-industries/zed/issues/5065
* https://github.com/zed-industries/zed/issues/5307

so far, other things (e.g. git and better AI support) are being worked on, and Zed team does not have any extra capacity to take that on.

Luckily, the community now can work on things independently (as Linux and Windows are mainly driven now) due to the source code being open.
So far though, there's been zero proposals on how to implement or approach this in Zed though.
